docs/configuration/CONFIGURATION_SUPPORTED_EXTRACTOR.md

```diff
@@ -40,48 +32,41 @@ For an up-to-date documentation, run `ng help @o3r/components:extractor`
       "@your/o3r-components"
     ]
   }
-},
+}
 ```
 
-__Note:__ This options will not search for the duplicate configurations in libraries.
+__Note:__ This option will not search for the duplicate configurations in libraries.
```

Comment:
What does it means? It will not put any messages or check if there are duplicates between the libraries?